Code/GraphMol/Depictor/RDDepictor.cpp

```diff
@@ -840,85 +845,84 @@ void generateDepictionMatching3DStructure(RDKit::ROMol &mol,
                                         25, true, forceRDKit);
 }
 
-void straightenDepiction(RDKit::ROMol &mol, int confId) {
+void straightenDepiction(RDKit::ROMol &mol, int confId, bool minimizeRotation) {
+  if (!mol.getNumBonds()) {
+    return;
+  }
   constexpr double RAD2DEG = 180. / M_PI;
   constexpr double DEG2RAD = M_PI / 180.;
-  constexpr double INCR_DEG = 30.;
-  constexpr double HALF_INCR_DEG = 0.5 * INCR_DEG;
-  constexpr double TOL_DEG = 5.0;
   constexpr double ALMOST_ZERO = 1.e-5;
   auto &conf = mol.getConformer(confId);
   auto &pos = conf.getPositions();
-  std::unordered_map<int, std::pair<unsigned int, double>> thetaBins;
-  std::vector<double> thetaValues;
-  thetaValues.reserve(mol.getNumBonds());
+  std::unordered_map<int, DepictorLocal::ThetaBin> thetaBins;
   for (const auto b : mol.bonds()) {
     auto bi = b->getBeginAtomIdx();
     auto ei = b->getEndAtomIdx();
     auto bv = pos.at(bi) - pos.at(ei);
     bv.x = (bv.x < 0.) ? std::min(-ALMOST_ZERO, bv.x)
                        : std::max(ALMOST_ZERO, bv.x);
     auto theta = RAD2DEG * atan(bv.y / bv.x);
-    auto d_theta = fmod(-theta, INCR_DEG);
-    if (fabs(d_theta) > HALF_INCR_DEG) {
-      d_theta -= DepictorLocal::copySign(INCR_DEG, d_theta, ALMOST_ZERO);
+    auto d_theta = fmod(-theta, 30.0);
```

Comment:
It seems like it would have been better to keep the constexpr definition of `INCR_DEG` rather than having the constant value 30.0 (or the dependent 15.0) repeated multiple times. Why did you make that change?

Reply:
You are probably right, even though I don't think it particularly clarifies that code. Done.